zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/controller/usermanagement/dto/AuthorizationAssignRequest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.gateway.rest.controller.usermanagement.dto;
+
+import io.camunda.zeebe.protocol.impl.record.value.authorization.AuthorizationOwnerType;
+import java.util.List;
+
+public class AuthorizationAssignRequest {
```

Comment:
❌ This DTO shouldn't exist. The request is autogenerated according to the OpenAPI spec in `rest-api.yaml`